src/app/[team]/scrim/[scrimId]/page.tsx

```diff
@@ -180,6 +185,11 @@ export default async function ScrimDashboardPage({ params }: Props) {
                     <CardHeader className="">
                       <h3 className="z-10 text-3xl font-semibold tracking-tight text-white">
                         {map.name}
+                        <CardFooter className="float-right flex items-center justify-between pt-10">
+                          <div className="z-10 font-semibold tracking-tight text-white">
+                            {map.replayCode ?? ""}
+                          </div>
+                        </CardFooter>
```

Comment:
This needs to be under the `CardContent`, this is appearing like this:
![CleanShot 2024-08-23 at 00 48 58@2x](https://github.com/user-attachments/assets/3efd9bbb-0aaf-4017-b438-caf93d1f530a)